troubleshooting/known_issues/ims_images_orphaned_in_s3.md

```diff
@@ -0,0 +1,242 @@
+# Known Issue: IMS Images Orphaned in S3
+
+There was an issue where image files larger than 5Gb were not correctly deleted from S3 when the image
```

Comment:
I notice the use of "was" here. If this was fixed at some point, we should specify when it was fixed (i.e. what version of either CSM or the relevant service -- and if we go with the service version, should tell them how to check the service version on their system).

Reply:
There is a fix checked into 1.6.1 and 1.7.0 and later. The fix does not clean up currently orphaned items, so this is still valid to run on these and later systems. I will add verbiage to that effect.

Reply:
Yeah, I thought that was the case. I mainly meant that if we start out by saying "There was a bug", then a natural thing one will wonder is when it was fixed.

Reply:
Fixed.